content/docs/user-guide/assets/pipeline/metadata.md

```diff
@@ -0,0 +1,85 @@
+description: In Open 3D Engine (O3DE), Metadata Asset Relocation allows files to be moved and renamed freely without breaking existing references by storing a UUID in a side-car file (.meta).
+weight: 600
+toc: true
+---
+
+Metadata Asset Relocation allows files to be moved and renamed freely without breaking existing references by storing a UUID in a side-car file (.meta).  See here for technical details on how the system works: https://github.com/o3de/sig-content/blob/main/rfcs/rfc-77-metadata-asset-relocation.md
+
+By default, the system is currently disabled as not all file types are currently able to be relocated without issue.  File types must be enabled individually.  Once a type is enabled, the Asset Processor will generate metadata files for every file of the given type on startup.  Any file which has a metadata associated with it can then be moved/renamed.  The Asset Processor does not need to be running when moving/renaming files and there are no special tools required to do so.
+
+## How to use
+Moving/renaming assets can be done either in the editor via the asset browser or using any file management tool of your choosing.
+
+There is only one requirement when **moving/renaming an asset: the metadata file must be moved/renamed as well**.
+
+Folders can be moved/renamed without having to touch metadata files, but note that all files within a folder must have a metadata file to ensure existing references do not break.
+
+Examples:
+
+* If blue.png is renamed to red.png, the metadata file blue.png.meta must be renamed to red.png.meta.
+* If textures/hat.png is moved to character/hat.png, the metadata file must be moved to character/hat.png.meta.
+
+
+If using source control, metadata files **must** be checked in along with the source asset and updated with any move/rename changes.
+
+There is one exception to the above, which is Intermediate Asset metadata files.  These metadata files follow the same rules as Intermediate Assets: they should not be modified or checked into source control.
+
+## How to Enable
+To enable a file type, create a .setreg file with the following settings:
+```json
+{
+    "O3DE":
+    {
+        "AssetProcessor":
+        {
+            "Settings":
+            {
+                "Metadata":
+                {
+                    "EnabledTypes": [
+                    ]
+                }
+            }
+        }
+    }
+}
+```
+EnabledTypes is simply an array of strings of file extension to enable.  For example, `".txt", ".png"`
+
+It is highly recommened that this setreg file be placed in a central location which is shared with other users of the project to ensure everyone is generating the same metadata files.  For a game project, it should be placed in the project directory; for gem development, it should be placed in the gem directory.  Keep in mind this is an Asset Processor wide setting, it will apply to all files the AP processes, not just the ones in the project/gem where the setting exists.

```

Comment:
```suggestion
It is highly recommended that this setreg file be placed in a central location, which is shared with other users of the project, to ensure everyone is generating the same metadata files.  For a game project, it should be placed in the project directory; for gem development, it should be placed in the gem directory.  Keep in mind this is an Asset Processor (AP) wide setting, it will apply to all files the AP processes, not just the ones in the project or gem where the setting exists.

```